### PR TITLE
Drop usage of backingStorePixelRatio API

### PR DESCRIPTION
**Pull request by mattpap**
This is unlikely to improve anything with regards to issue #8478, but it's a good thing to do anyway.

addresses #8478

### CODE REVIEW

File: bokehjs/src/lib/models/canvas/canvas.ts [hunk 1]
```diff
@@ -114,13 +114,13 @@ export class CanvasView extends DOMView {
...
   prepare_canvas(width: number, height: number): void {
-    // Ensure canvas has the correct size, taking HIDPI into account
     this.bbox = new BBox({left: 0, top: 0, width, height})
 
     this.el.style.width = `${width}px`
     this.el.style.height = `${height}px`
 
-    const pixel_ratio = get_scale_ratio(this.ctx, this.model.use_hidpi, this.model.output_backend)
+    const {use_hidpi, output_backend} = this.model
+    const pixel_ratio = use_hidpi && output_backend != "svg" ? devicePixelRatio : 1
```
Comment (bryevdv) [inline bokehjs/src/lib/models/canvas/canvas.ts]: Where is the value of `devicePixelRatio` coming from?

Comment (mattpap) [reply bokehjs/src/lib/models/canvas/canvas.ts]: `window`, or the global scope in general. Given that we don't rely on jsdom and nodejs for testing anymore, we can refer to all globals equally.

**Comment by mattpap** [comment]
The test failure is unrelated, so I won't be restarting CI.

File: bokehjs/src/lib/models/canvas/canvas.ts (hunk 1 above)
Comment (bryevdv) [reply bokehjs/src/lib/models/canvas/canvas.ts]: What specifically has changed that we can drop the old method of computing the pixel ratio?

Comment (mattpap) [reply bokehjs/src/lib/models/canvas/canvas.ts]: `backingStorePixelRatio` was never part of canvas specification, so it shouldn't have been used in the first place. Its usage is deprecated. As far as I can tell, if it was available at all (as a prefixed property), its value was usually 1, thus not contributing anything. Perhaps some older variants of some browsers (Safari?) messed with this setting to some extent.

**Comment by bryevdv** [comment]
I'll test locally on osx tonight then merge

**Comment by bryevdv** [comment]
Looks good on FF, Chrome, and Safari on OSX. Will test on windows with rc1

**Pull request merged**